src/passes/Asyncify.cpp

```diff
@@ -455,8 +452,45 @@ class ModuleAnalyzer {
 
     map.swap(scanner.map);
 
+    std::set<Name> blacklist_names;
+    std::set<Name> whitelist_names;
+    std::set<std::string> blacklist_patterns;
+    std::set<std::string> whitelist_patterns;
+    std::set<std::string> blacklist_patterns_matched;
+    std::set<std::string> whitelist_patterns_matched;
```

Comment:
Our naming convention would be `whitelistPatternsMatched` etc

Reply:
Done.